orchestrator/src/main/kotlin/Orchestrator.kt

```diff
@@ -395,12 +417,15 @@ class Orchestrator(
     }
 
     /**
-     * Publish a message to the [ConfigEndpoint] and update the current ORT run to the active state.
+     * Publish a message with the given [header] to the [ConfigEndpoint] to trigger the Config worker job for the given
+     * [run]. If [updateRun] is *true*, set the status of the run to ACTIVE.
      */
-    private fun scheduleConfigWorkerJob(run: OrtRun, header: MessageHeader) {
+    private fun scheduleConfigWorkerJob(run: OrtRun, header: MessageHeader, updateRun: Boolean) {
```

Comment:
I am a bit puzzled by this change.

Why make this a parameter ?
In your PR, it `seems` your are always calling it with `true`.

Reply:
When a lost schedule is detected, the function is invoked with `updateRun = false`. It is then not necessary to update the database; the run is already in ACTIVE state.